docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -763,6 +763,42 @@ Default value: 10000.
 <max_thread_pool_size>12000</max_thread_pool_size>
 ```
 
+## max_thread_pool_free_size {#max-thread-pool-free-size}
+
+The number of threads that are free in the Global Thread pool.
```

Comment:
Better to say that this is a number of threads that global pool holds all the time even if there are no compute load. Other threads could be released to OS and then allocated from it once again.